plugins/unifying/fu-unifying-hidpp.c

```diff
@@ -0,0 +1,265 @@
+/*
+ * Copyright (C) 2016-2018 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-unifying-common.h"
+#include "fu-unifying-hidpp.h"
+
+#if 0
```

Comment:
Probably worth leaving a comment here why this is `#if 0`'ed out.

Reply:
I think just nuke it; we have git to preserve history. It was only really useful for doing the initial work with signed bootloaders.